include/quicly.h

```diff
@@ -332,32 +332,54 @@ struct st_quicly_conn_streamgroup_state_t {
+         */                                                                                                                        \
         uint64_t lost;                                                                                                             \
+        /**                                                                                                                        \
+         * Total number of packets for which an acknowledgement has been received.                                                 \
+         */                                                                                                                        \
         uint64_t ack_received;                                                                                                     \
+        /**                                                                                                                        \
+         * Total number of packets for which an acknowledgement was received after they were marked lost.                          \
+         */                                                                                                                        \
         uint64_t late_acked;                                                                                                       \
     } num_packets;                                                                                                                 \
     struct {                                                                                                                       \
         /**                                                                                                                        \
-         * This value is calculated at UDP datagram-level, and used for determining the amplification limit.                       \
+         * Total bytes received, at UDP datagram-level. Used for determining the amplification limit.                              \
          */                                                                                                                        \
         uint64_t received;                                                                                                         \
+        /**                                                                                                                        \
+         * Total bytes sent, at UDP datagram-level.                                                                                \
+         */                                                                                                                        \
         uint64_t sent;                                                                                                             \
     } num_bytes
 
 typedef struct st_quicly_stats_t {
+    uint8_t stats_version;
```

Comment:
I thought we agreed to remove this field?

Reply:
Gah! Yes, removing.